pkg/admin/admin.go

```diff
@@ -58,21 +47,5 @@ func (h *handler) servePing(w http.ResponseWriter, req *http.Request) {
 }
 
 func (h *handler) serveReady(w http.ResponseWriter, req *http.Request) {
-	if h.getReady() {
-		w.Write([]byte("ok\n"))
-	} else {
-		http.Error(w, "unready", http.StatusServiceUnavailable)
-	}
-}
-
-func (h *handler) getReady() bool {
-	h.RLock()
-	defer h.RUnlock()
-	return h.ready
-}
-
-func (h *handler) setReady(ready bool) {
-	h.Lock()
-	defer h.Unlock()
-	h.ready = ready
+	w.Write([]byte("ok\n"))
```

Comment:
Hmm, yeah, I thought about that, but it would require changing our install yaml. I figured I'd leave it as is in case we ever wanted to add some other additional readiness requirement in the future.